### PR TITLE
Port all apps to use argparse

### PR DESCRIPTION
**Pull request by infirit**
For the logging module we need to be able to set the log level from the command line.

### CODE REVIEW

File: apps/blueman-adapters.in [hunk 1]
```diff
@@ -19,22 +20,21 @@ _dirname = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
...
 from blueman.main.Adapter import BluemanAdapters
 
 # Workaround introspection bug, gnome bug 622084
 signal.signal(signal.SIGINT, signal.SIG_DFL)
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--socket-id", dest="socket_id", action="store", type=int, metavar="ID")
+    parser.add_argument("adapter", nargs="*", metavar="ADAPTER NAME")
```
Comment (cschramm) [inline apps/blueman-adapters.in]: `nargs="*"`? I think that allows multiple values and returns a list which `BluemanAdapters` does not expect.

File: apps/blueman-rfcomm-watcher.in [hunk 2]
```diff
@@ -18,10 +19,14 @@ def on_file_changed(monitor, file, other_file, event_type):
     if event_type == Gio.FileMonitorEvent.DELETED:
         loop.quit()
 
-mon = Gio.File.new_for_path(sys.argv[1]).monitor_file(Gio.FileMonitorFlags.NONE)
+parser = argparse.ArgumentParser()
+parser.add_argument("path", dest="path", nargs="+", action="store")
```
Comment (cschramm) [inline apps/blueman-rfcomm-watcher.in]: Same here: `nargs="+"` allows multiple values and returns a list, but `os.open` expects a single value.

File: apps/blueman-sendto.in [hunk 3]
```diff
@@ -135,6 +115,19 @@ class SendTo:
...
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-d", "--device", dest="device", action="store",
+                        help=_("Send files to this device"), metavar="ADDRESS")
+    parser.add_argument("--dest", dest="device", action="store",
+                        help=_("Same as --device"), metavar="ADDRESS")
+    parser.add_argument("-s", "--source", dest="source", action="store",
+                        help=_("Source adapter. Takes address or adapter's name eg. hci0"), metavar="PATTERN")
+    parser.add_argument("-u", "--delete", dest="delete", action="store_true", help=_("Delete files on exit"))
+    parser.add_argument("files", nargs="*", action="append", metavar="FILE",
```
Comment (cschramm) [inline apps/blueman-sendto.in]: Why `action="append"`? I don't think that has an effect to the usage you're just using the first bunch anyway (`parsed_args.files[0]`).

File: apps/blueman-adapters.in (hunk 1 above)
Comment (infirit) [inline apps/blueman-adapters.in]: I'll change it to ``nargs="?"``, only one value read

File: apps/blueman-rfcomm-watcher.in (hunk 2 above)
Comment (infirit) [inline apps/blueman-rfcomm-watcher.in]: I'll drop both dest and nargs here making it mandatory to provide something

**Pull request closed without merging**